src/tools/antool/antool

```diff
@@ -33,8 +33,8 @@
 import argparse
 from os import stat
 
-from listsyscalls import *
-from syscalltable import *
+from antoollib.listsyscalls import *
```

Comment:
IMHO libantool would look better, but I don't have a strong opinion here.

Reply:
In this place maybe yes, but what about the path, what is better:
```/usr/lib/python3.5/site-packages/antoollib/```
or
```/usr/lib/python3.5/site-packages/libantool/``` ?
Moreover ```antoollib``` is not a library, but helper modules for antool, so ```libantool``` could be misleading.

Reply:
... so I am not sure that 'libantool' is really better

Reply:
Ok, I looked how other Python applications are packaged and it seems they don't install their files to /usr/lib/python*/site-packages/. What they do is put all python files in /usr/share/$appname/ and install small shell script in /usr/bin that calls python script in /usr/share/$appname/.